aroma/tests/data/integration_test_ground_truth/classified_motion_ICs.txt

```diff
@@ -0,0 +1 @@
+3,5,6,12,13,14,15,17,21,24,28
```

Comment:
Are these ICs always the same? When we didn't do any smoothing I saw some of these would always be selected but some other not. My guess is that for instance `21`and `22` were very close in terms of where they fall in the ICA results, and so sometimes they would be "flipped", meaning that `21` here could be `22` when running the code again.